setup.py

```diff
@@ -2,7 +2,7 @@
 
 from setuptools import setup
 
-VERSION = '0.4.11'
+VERSION = '0.5.0'
```

Comment:
@benpatterson or @jzoldak -- do you want this version change to be in a different PR? I believe there is an email thread about the next version.